ui2/src/components/InputDialog/InputDialog.tsx

```diff
@@ -87,6 +88,7 @@ export function InputDialog<T extends { [key: string]: any }>(props: InputDialog
         <Button className="ghost" onClick={() => props.onClose(null)}>
           Cancel
         </Button>
+        {props.error && <p className="error">{props.error}</p>}
```

Comment:
we could add in `AppError` to this code at some point as well

Reply:
agreed, reintroducing the standard error handling we had before is going to be a future pr for sure :)